oak_abi/src/label/mod.rs

```diff
@@ -54,6 +54,25 @@ impl crate::proto::oak::label::Label {
         }
     }
 
+    /// Returns a a WebAssembly module SHA-256 hash label.
+    pub fn hash(hash: &[u8]) -> Self {
+        let hash_tag = web_assembly_module_tag(&hash);
+        Label {
+            confidentiality_tags: vec![hash_tag],
+            integrity_tags: vec![],
+        }
+    }
+
+    /// Returns a WebAssembly module Ed25519 public key label.
+    /// https://ed25519.cr.yp.to
+    pub fn public_key(public_key: &[u8]) -> Self {
+        let public_key_tag = web_assembly_module_signature_tag(&public_key);
+        Label {
+            confidentiality_tags: vec![public_key_tag],
+            integrity_tags: vec![],
+        }
+    }
+
```

Comment:
I don't think these methods should be here (or exist at all). AFAICT they are just wrapping a tag into a label by using it as confidentiality component only. How about a `confidentiality_label` method (outside of `Label`, as top-level, like the existing `web_assembly_module_signature_tag` & similar methods). Also the naming is quite vague, and inconsistent with the existing methods about tags, or is there a reason for that?

Reply:
I thought since we already have `public_untrusted` method, we just could add convenience functions for simple labels.

Reply:
`public_untrusted` is like `default` or `empty`, that's why it is defined on `Label`. But I think individual label constructors can just be regular functions outside of it.

Reply:
Done, added:
```rust
/// Convenience function for creating labels with a single confidentiality tag.
pub fn confidentiality_label(tag: Tag) -> crate::proto::oak::label::Label {
    Label {
        confidentiality_tags: vec![tag],
        integrity_tags: vec![],
    }
}
```